testserver/test/acceptance/pages/login_page.py

```diff
@@ -0,0 +1,26 @@
+__author__ = 'Kashif'
+
+"""
+Login page for the Mckinsey Site.
+"""
+
+from bok_choy.page_object import PageObject
+from . import base_url
+
+
+class LoginPage(PageObject):
+
+    url = base_url
+
+    def is_browser_on_page(self):
+        #check the browser title
```

Comment:
Although we don't run pylint/pep8 on these files, it'd be good to follow standard conventions, like docstrings on every method:

def foo(self):
    """
    comment here
    """